app/src/main/java/org/mozilla/focus/fragment/FirstrunFragment.java

```diff
@@ -0,0 +1,54 @@
+/* -*- Mode: Java; c-basic-offset: 4; tab-width: 20; indent-tabs-mode: nil; -*-
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+package org.mozilla.focus.fragment;
+
+import android.content.Context;
+import android.os.Bundle;
+import android.preference.PreferenceManager;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+
+import org.mozilla.focus.R;
+import org.mozilla.focus.activity.MainActivity;
+
+public class FirstrunFragment extends Fragment implements View.OnClickListener {
+    public static final String FRAGMENT_TAG = "firstrun";
+
+    private static final String FIRSTRUN_PREF = "firstrun_shown";
+
+    public static FirstrunFragment create() {
+        return new FirstrunFragment();
+    }
+
+    public static boolean shouldShowFirstrun(final Context context) {
+        return !PreferenceManager.getDefaultSharedPreferences(context).getBoolean(FIRSTRUN_PREF, false);
+    }
```

Comment:
I recently introduced the `Settings` class for such helpers. Maybe you want to move it there.